data/topic-schema/src/main/resources/net/corda/schema/Flow.yaml

```diff
@@ -3,6 +3,7 @@ topics:
     name: flow.event
     consumers:
       - flow
+      - db
```

Comment:
We shouldn't need the DB worker to consume from the flow topics

Reply:
Yeah, you are right. I missed the direction of the arrows in the diagram.